tests/tools/combine-tools.test.js

```diff
@@ -0,0 +1,266 @@
+const fs = require('fs');
+const path = require('path');
+const { combineTools } = require('../../scripts/tools/combine-tools');
+const {
+  expectedDataT1,
+  manualToolsWithMissingData,
+  manualToolsToSort,
+  automatedToolsT5,
+  automatedToolsT4,
+  manualToolsT4,
+  automatedToolsT6,
+  automatedToolsT7,
+  automatedToolsT8,
+  manualToolsT8,
+  automatedToolsT9,
+  manualToolsT9,
+  automatedToolsT12,
+  invalidAutomatedToolsT10,
+  manualToolsWithInvalidURLT11,
+  circularTool
+} = require('../fixtures/combineToolsData');
+
+jest.mock('ajv', () => {
+  return jest.fn().mockImplementation(() => ({
+    compile: jest.fn().mockImplementation(() => (data) => data.title !== 'Invalid Tool'),
+  }));
+});
```

Comment:
@vishvamsinh28 Why is there a need to mock `ajv` here? 	As far as I know, it is used to validate the object against a json schema, and we should do that during test execution.

Reply:
There is an if block for isValid that was left uncovered, so I mocked AJV to get inside that block, cover the lines, and test it.